### DISCUSSION TITLE
Grand Coding Contest - Google Developer Students Club & Newton School

### DISCUSSION THREAD

**Discussion by Yashkapure06** [Yashkapure06/GDSC-ProCode#495] [Announcements]
### Hello Everyone do register for this coding contest and give your 100% efforts.

# Grand Coding Contest: Google Developer Students Club - MET & Newton School 

<img align="center" src="https://user-images.githubusercontent.com/61585443/197405721-9e22b49e-b738-442a-9a05-b5d7c0fe2e21.jpeg" width="500" height="500"/>

### **| 𝐏𝐫𝐢𝐳𝐞 𝐌𝐨𝐧𝐞𝐲 & 𝐆𝐢𝐟𝐭 𝐕𝐨𝐮𝐜𝐡𝐞𝐫𝐬 𝐰𝐨𝐫𝐭𝐡- ₹ 30,000 | 𝐅𝐫𝐞𝐞 𝐑𝐞𝐠𝐢𝐬𝐭𝐫𝐚𝐭𝐢𝐨𝐧 |💰**

   **💡 REGISTER NOW** - [**STRICTLY USE this Registration Link ❗**](https://bit.ly/October-Coding-Contest-MET)

- Unlock the _**Free Exclusive**_ Beginner Friendly Courses 
- Language Used: **c / c++ / Java / Javascript / Python** 
- Date of Contest: 27th Oct 2022 (Thursday) at 9:00 pm - 11:30 pm
- Eligibility: **_Open to All_**

**Perks** : 
         **_Individual Event_**
         1. 1st rank holder will receive **Swags**
         _**Team Event**_
         1. 1st rank holder team will receive **Swags**. (NOTE: Every member in team will receive swags)
          

Join Our Community - [**Discord**](https://bit.ly/3wug972)

PS: **Unique Questions Designed By ICPC Finalist**

<video src="https://user-images.githubusercontent.com/61585443/197406215-04655658-bb4d-4a51-ad0b-c1f7b3e930ce.mp4" width="500" height="800"></video>

# NOTE!!! : Whoever registers over here will have to comment below that you have registered! 

``` Thank You🧡 ```

**Comment by AtharvaKawale124** [comment]
Registered.

**Reply by Yashkapure06** [reply]
Thank You @AtharvaKawale124 
Ask your friends from same college or from different college to do the same!

**Comment by rajashri1626** [comment]
@Yashkapure06  I have registered!

**Comment by Gaurav-137** [comment]
**@Yashkapure06**  I have registered successfully and very excited for the contest.

**Reply by Yashkapure06** [reply]
Thank You @Gaurav-137 
Ask your friends from same college or from different college to do the same!

**Reply by Gaurav-137** [reply]
Definitely..

**Comment by paras-kitey** [comment]
Registered

**Reply by Yashkapure06** [reply]
Thank You @paras-kitey 
Ask your friends from same college or from different college to do the same!

**Comment by rushirao17** [comment]
Registered...

**Reply by Yashkapure06** [reply]
Thank You @rushirao17 
Ask your friends from same college or from different college to do the same!

**Comment by chinmay444** [comment]
Heyyy!! I have successfully registered Excited for contest_😇

**Reply by Yashkapure06** [reply]
Thank You @chinmay444 
Ask your friends from same college or from different college to do the same!

**Comment by sejals2** [comment]
I have registered

**Reply by Yashkapure06** [reply]
Thank You @sejals2 
Ask your friends from same college or from different college to do the same asps!

**Comment by janhvimali** [comment]
Registered successfully

**Reply by Yashkapure06** [reply]
Thank You @janhvimali 
Ask your friends from same college or from different college to do the same asps!

**Comment by mohee1509** [comment]
Registered

**Reply by Yashkapure06** [reply]
Thank you @mohee1509 
Ask your friends from same college or from different college to do the same asps!

**Comment by pickl3l0ve** [comment]
![Screenshot (65)](https://user-images.githubusercontent.com/116171787/197488712-b5ce67e8-58a7-4ff7-9c48-5d9f6fc21549.png)
@Yashkapure06

**Reply by Yashkapure06** [reply]
Thank You @maxmiller999 
Ask your friends from same college or from different college to do the same asps!

**Comment by kunalnandre** [comment]
Registered successfully  and very excited for the Contest.

**Comment by devarshishimpi** [comment]
Registered @Yashkapure06

**Reply by Yashkapure06** [reply]
Thanks Dude <3
